databases/week1/sql-homework-week1.sql

```diff
@@ -0,0 +1,85 @@
+SELECT
+  count(*)
+FROM
+  task;
+--task1--
+SELECT
```

Comment:
Task description: "Find out **how many** tasks in the task table do not have a valid due date"
This query brings me back 8 rows with NULL due date. Try to find a way to bring a number instead of selecting all the ones that has a due_date = null